packages/devtools_app/lib/src/debugger/flutter/debugger_screen.dart

```diff
@@ -372,14 +368,23 @@ class ScriptPickerState extends State<ScriptPicker> {
 
   Widget _buildScript(ScriptRef ref) {
     final selectedColor = Theme.of(context).selectedRowColor;
+
+    TextStyle style;
+    if (ref == widget.selected) {
+      style = TextStyle(color: Theme.of(context).textSelectionColor);
+    }
+
     return Material(
       color: ref == widget.selected ? selectedColor : null,
       child: InkWell(
         onTap: () => widget.onSelected(ref),
         child: Container(
           padding: const EdgeInsets.all(4.0),
           alignment: Alignment.centerLeft,
-          child: Text('${ref?.uri?.split('/')?.last} (${ref?.uri})'),
+          child: Text(
+            '${ref?.uri?.split('/')?.last} (${ref?.uri})',
+            style: style,
```

Comment:
nit: why not inline this like it is done for `color`? (e.g. `ref == widget.selected ? TextStyle(color: Theme.of(context).textSelectionColor) : null`)